product-service/handlers/common/helpers.ts

```diff
@@ -0,0 +1,19 @@
+export function buildResponse(data: any, statusCode: number = 200): Response {
+    return {
+        statusCode,
+        headers: {
+            'Access-Control-Allow-Origin': '*',
+            'Access-Control-Allow-Credentials': true,
+        },
+        body: JSON.stringify(data)
+    }
+}
+
+interface Response {
+    headers: {
```

Comment:
It is better to set the type like `headers: Record<string, string | boolean>` or save headers to enum and specify the type here